src/App.svelte

```diff
@@ -1,7 +1,12 @@
 <script>
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
-  console.log(exampleData);
+  export let grabIP = function(text){
+    const ips = document.getElementsByClassName('ip');
+    const toCopy = Array.from(ips).filter((ip) => {return ip.value === text})[0];
+    toCopy.select();
+    document.execCommand("copy");
+  }
```

Comment:
Why did you remove navigator usage? Why not have a fallback?

Reply:
I don't think if this fails the nav is gonna work.

Reply:
happy to be wrong though, feel free to link some documentation that suggests otherwise.

Reply:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard